docs/resources/User.md

```diff
@@ -14,6 +14,22 @@ data:image/jpeg;base64,BASE64_ENCODED_JPEG_IMAGE_DATA
 
 Ensure you use the proper header type (`image/jpeg`, `image/png`, `image/gif`) that matches the image data being provided.
 
+## Usernames and Nicknames
+
+Discord enforces the following restrictions for usernames and nicknames:
```

Comment:
> usernames **and nicknames**

Items 3 and 4 currently do not apply to nicknames (and the list uses the ambiguous term "Names"). Is this intentional, or should it otherwise be clarified?